### PR TITLE
Paul Tols Named colors

### LINKED ISSUE #20: This project still has a REQUIRE file - should we update?
I started typing in the first few colours and wanted to test locally, but it seems in Julia 1.6 I am not able to set it to dev mode?
Should I introduce a Project.toml maybe?

### PR DESCRIPTION
**Pull request by kellertuer**
This PR introduces the colours from Paul Too (excluding the colormaps of course)

https://personal.sron.nl/~pault/

and switches the package to using a Project.toml.
This resolves #20 and #18.

### CODE REVIEW

**Comment by kellertuer** [comment]
This PR should be ready to merge. Can you maybe check @oxinabox ?

File: data/paul-tol.txt
Comment (oxinabox) [inline data/paul-tol.txt]:
```suggestion
; transcribed to NamedColors.jl by Ronny Bergmann (kellertuer).
```

**Comment by oxinabox** [comment]
This looks good, before we can merge need to understand the copyright and liscensing of this color list.
And add that to the top of the data file.

**Comment by kellertuer** [comment]
Well, Paul has a copyright on the page (see footer) but I wrote im an email mit of November, which he answered with

```
Hi Ronny,

Nice to hear my schemes are so useful.
You can add it to Julia, adding my name and the website url.
Some schemes are copied or adapted from ColorBrewer, so take that into account if you add those.

Best regards, Paul
```

**Comment by oxinabox** [comment]
awesome. thanks. :heart:

**Review by oxinabox — APPROVED** [review]

**Comment by oxinabox** [comment]
I have sent you an invite to be a repo maintainer.
Please follow [ColPrac](https://github.com/SciML/ColPrac)

**Comment by kellertuer** [comment]
Wow, super! Sure I accepted and I am happy to help and will follow ColPrac.

**Pull request merged**
